jsconcat.php

```diff
@@ -203,7 +203,8 @@ function do_items( $handles = false, $group = false ) {
 					}
 				}
 				if ( isset( $href ) ) {
-					printf( '<script type="text/javascript" src="%s" %s></script>', $href, $attr_string );
+					$script_html = sprintf( '<script type="text/javascript" src="%s" %s></script>', $href, $attr_string );
+					echo apply_filters( "ngx_http_concat_script_loader_tag",  $script_html, $href, $attr_string );
```

Comment:
Naming nitpick. It seems that the `ngx_http_concat` is used for more global (CSS and js) filters while `js_concat` is used for js-specific stuff such as this.
```suggestion
					echo apply_filters( "js_concat_script_loader_tag",  $script_html, $href, $attr_string );
```

Reply:
Oh nevermind I had a peak into more info you send, and I can see this is a pair filter to `ngx_http_concat_style_loader_tag` so the naming absolutely makes sense as it is.

Reply:
yes, forgot to mention directly the naming choice...